uint64 uid                      # Unique identifier
geometry_msgs/Point center      # Central point [m]
geometry_msgs/Vector3 velocity  # Linear velocity [m/s]
float64 radius                  # Radius with added margin [m]
float64 true_radius             # True measured radius [m]
string semclass                  # Semantic class
float64 confidence              # Confidence in semantic class
